core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/ShuffledHashJoinExecParser.scala

```diff
@@ -46,6 +48,7 @@ case class ShuffledHashJoinExecParser(
 
     // TODO - add in parsing expressions - average speedup across?
     ExecInfo(node, sqlID, node.name, "", speedupFactor,
-      maxDuration, node.id, isSupported, None)
+      maxDuration, node.id, isSupported, None, execsRef = execNameRef,
```

Comment:
nit: Since we are touching this anyway, can we make  them all named parameter in all the parser classes? :) It makes reading the code much easier. No pressure to do it though.